src/content.config.ts

```diff
@@ -0,0 +1,24 @@
+import { z, defineCollection } from "astro:content";
```

Comment:
Part of the conversion to the new Content Layer API, see https://docs.astro.build/en/guides/upgrade-to/v5/#updating-existing-collections